modules/dns_response_policy/README.md

```diff
@@ -0,0 +1,96 @@
+# gcp_dns_response_policy
+
+This module is used to create DNS response policy and rules to the corresponding response policy.
+
+Cloud DNS private zones let you create a single response policy per network that modifies resolver behavior according to the policy created.
+
+You can create multiple rules within one response policy with following behaviors:
+
+- **Default / Unspecified**: Alter results for selected query names (including wildcards) by providing specific resource records.
+- **Bypass**: Trigger [passthru](https://datatracker.ietf.org/doc/id/draft-vixie-dnsop-dns-rpz-00.html#rfc.section.3.3) behavior that bypasses the response policy, exempting names that would otherwise match.
+
+Reference: https://cloud.google.com/dns/docs/zones/manage-response-policies
+
+## Usage
+
+Basic usage of this module for a private zone is as follows:
+
+```hcl
+module "dns_response_policy" {
+  source             = "terraform-google-modules/cloud-dns/google//modules/dns_response_policy"
+  project_id         = "example-project-id"
+  policy_name        = "example-policy-name"
+  network_self_links = ["projects/example-project-id/global/networks/default"]
+  description        = "example-dns-response-policy"
+  rules = {
+    "override-google-com" = {
+      dns_name = "*.google.com."
+      rule_local_datas = {
+        "A" = { # Record type.
+          rrdatas = ["192.0.2.91"]
+          ttl     = 300
+        },
+        "AAAA" = {
+          rrdatas = ["2001:db8::8bd:1002"]
+          ttl     = 300
+        }
+      }
+    },
+    "override-withgoogle-com" = {
+      dns_name = "withgoogle.com."
+      rule_local_datas = {
+        "A" = {
+          rrdatas = ["193.0.2.93"]
+          ttl     = 300
+        }
+      }
+    },
+    "bypass-google-account-domain" = {
+      dns_name      = "account.google.com."
+      rule_behavior = "BYPASS"
+    }
+  }
+}
+```
+
+## Requirements
+
+| Name | Version |
+|------|---------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 0.13 |
+| <a name="requirement_google-beta"></a> [google-beta](#requirement\_google-beta) | ~> 4.0 |
+
+## Providers
+
+| Name | Version |
+|------|---------|
+| <a name="provider_google-beta"></a> [google-beta](#provider\_google-beta) | ~> 4.0 |
+
+## Modules
+
+No modules.
+
+## Resources
+
+| Name | Type |
+|------|------|
+| [google-beta_google_dns_response_policy.this](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/google_dns_response_policy) | resource |
+| [google-beta_google_dns_response_policy_rule.this](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/google_dns_response_policy_rule) | resource |
+
+## Inputs
```

Comment:
let's wrap these in the doc hooks
https://github.com/terraform-google-modules/terraform-google-cloud-dns/blob/c0f39e462d55adbb635ea36c260085eae58cf1b9/README.md?plain=1#L58-L89